.gitignore

```
@@ -0,0 +1 @@
+node_modules
```

```suggestion
node_modules

```
гітхаб рухається, коли в файлах в кінці немає порожнього рядка